sample/src/NimblePros.SampleToDo.UseCases/Projects/Create/CreateProjectHandler.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace NimblePros.SampleToDo.UseCases.Projects.Create;
 
-public class CreateProjectHandler : ICommandHandler<CreateProjectCommand, Result<int>>
+public class CreateProjectHandler : ICommandHandler<CreateProjectCommand, Result<ProjectId>>
```

Comment:
Again, I think it's nicer to see the domain terms (project ID) rather than the int in use-cases.